_posts/2022-09-13-#1689.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: pr
+date: 2022-09-13
+title: "Expose new BumpChannelClose event for channels with anchor outputs"
+authors: ["wpaulino"]
+pr: 1689
+components: ["tests"]
+host: ariard
+status: future
```

Comment:
This should be `upcoming` instead of future for the template logic to work.

Reply:
Yep, updated to `upcoming`.